discovery_child_development/pipeline/labelling/taxonomy_v2/taxonomy_labels.py

```diff
@@ -0,0 +1,211 @@
+"""
+This script is used to test the relevancy labelling prompt and function.
+
+Example usage:
+    python discovery_child_development/pipeline/labelling/taxonomy_v2/taxonomy_labels.py --dataset openalex --num_samples 5 --model gpt-3.5-turbo-1106 --output_filename testing
+"""
+import argparse
+import json
+import pandas as pd
+import asyncio
+import time
+from pathlib import Path
+
+from nesta_ds_utils.loading_saving import S3
+from discovery_child_development import logging, PROJECT_DIR, S3_BUCKET, get_yaml_config
+from discovery_child_development.getters import get_dataset
+from discovery_child_development.utils.openai_utils import (
+    MessageTemplate,
+    Classifier,
+    FunctionTemplate,
+)
+from discovery_child_development.utils.labelling_utils import (
+    create_category_description_string,
+)
+from discovery_child_development.utils.utils import (
+    create_directory_if_not_exists,
+    batch,
+)
+from discovery_child_development.utils import taxonomy_labelling_utils as tlu
+from discovery_child_development.utils.jsonl_utils import load_jsonl
+
+# Get labelling config params
+CONFIG = get_yaml_config(Path(__file__).resolve().parent / "config.yaml")
+# Define paths to the prompt and function definitions
+PATH_TO_PROMPTS = Path(__file__).resolve().parent / "prompts"
+PATH_TO_CATEGORIES = PATH_TO_PROMPTS / "categories.json"
+PATH_TO_MESSAGE_PROMPT = PATH_TO_PROMPTS / "prompt.json"
+PATH_TO_FUNCTION = PATH_TO_PROMPTS / "function.json"
+# PATH_TO_EXAMPLES = PATH_TO_PROMPTS / "examples.jsonl"
+# Define paths to the outputs
+OUTPUT_FILEPATH = PROJECT_DIR / CONFIG["local_output_directory"]
+
+
+async def main(
+    dataset: str,
```

Comment:
I think it's better to get rid of the dataset argument because this script will only work with the dataset `taxonomy_labelling_sample` - that's the only one that contains a `source` field

Reply:
I'm tempted to leave it as I guess we might have other datasets later down the line